src/components/Global/Header/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import Image from 'next/image'
+
+import logo from './logo.png';
+import styles from './Header.module.scss';
+
+type HeaderProps = {
+    profilePicture?: string;
+}
+
+const Header = ({ profilePicture }: HeaderProps) => {
+    const [activeTab, setActiveTab] = React.useState<0 | 1>(0);
+
+    return (
+        <div className={ styles.header }>
+            <div className={ styles.content}>
+                <div className={ styles.left }>
+                    <>
+                        <Image src={ logo } width={57} height={57} /> { /** This is a mock, change when there's logo component  */ }
+                        <p>Computational Puzzles for Kids</p>
+                    </>
+                    <div className={ styles.tabs }>
+                        <span className={ activeTab === 0 && styles.active } onClick={() => setActiveTab(0)}>
+                            <p>Puzzles Map</p>
+                        </span>
+                        <span className={ activeTab === 1 && styles.active } onClick={() => setActiveTab(1)}>
+                            <p>Puzzles List</p>
+                        </span>
+                    </div>
+                </div>
+                <div className={ styles.right }>
+                    <button> Profile </button>
+                    {   profilePicture
+                        ? <img  className={ styles.profileImg } src={ profilePicture } alt="profile" />
+                        : <span className={ styles.profileImg }> </span>
```

Comment:
would be cool to add text inside the profile image (like the first letter of their username/email) if they don't have one so there is at least something there

Reply:
But this could just be another feature and doesn't need to be in this one